apps/portlet-documents/src/main/webapp/vue-app/common/components/ExoDocument.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <v-list-item @click.prevent="openPreview()">
-    <v-list-item-icon class="mx-0">
-      <v-icon :color="documentIcon.color" x-large>
+    <v-list-item-icon class="mx-1">
+      <v-icon
+        :color="documentIcon.color"
+        size="35">
         {{ documentIcon.ico }}
```

Comment:
Shouldn't be renamed to iconObject for example with properties: iconObject.icon, iconObject.color ?